File: UI/src/app/_models/DTO/JobDTO.ts
export interface JobDTO {
  jobId: number;
  categoryId: number;
  cityId: number;
  title: string;
  category: string;
  typeJobId: number,
  employer: string;
  city: string;
  publishedOn: Date;
  finishedOn: Date;
  imagePath: string;
  contact: string;
  salary: string;
  description: string;
  numberOfApplicants: number;
  marked:number;// ?? boolean
}

File: JobSolution/JobSolution.DTO/DTO/JobDTO.cs
using JobSolution.Domain.Entities;
using System;
using System.Collections.Generic;
using System.IO;
using System.Text;

namespace JobSolution.DTO.DTO
{
    public class JobDTO 
    {
        public int JobId { get; set; }
        public int CategoryId { get; set; }
        public int CityId { get; set; }
        public int TypeJobId { get; set; }
        public string Title { get; set; }
        public string Category { get; set; }
        public string Employer { get; set; }
        public string City { get; set; }
        public DateTime? PublishedOn { get; set; }
        public DateTime? FinishedOn { get; set; }
        public DateTime? EnrolledDate { get; set; }
        public string ImagePath { get; set; }
        public string Contact { get; set; }
        public float Salary { get; set; }
        public string Description { get; set; }
        public string TypeJob { get; set; }
    }
}
